### PR TITLE
Fix #3443: Add GitHub star button to sandbox

### LINKED ISSUE #3443: Add GitHub star button to sandbox
**Feature**
Add feature issue reference

**Describe the task**

### PR DESCRIPTION
**Pull request by darth-coder00**
### Describe your changes :
Closes #3443 

I worked on the adding `Star us on Github` popover button for sandbox instances

#
### Type of change :

- [x] New feature

#
### Frontend Preview (Screenshots) :
<p>
On Landing page
Screenshot 2022-03-23 at 6 50 54 PM

On Landing page with feature modal open
Screenshot 2022-03-23 at 6 51 12 PM

</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@harshach @vivekratnavel @Sachin-chaurasiya @ShaileshParmar11

### CODE REVIEW

**Review by ShaileshParmar11 — APPROVED** [review]
LGTM!

File: openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.tsx [hunk 1]
```diff
@@ -498,3 +498,10 @@ export const getRandomColor = (name: string) => {
 export const isUrlFriendlyName = (value: string) => {
   return !UrlEntityCharRegEx.test(value);
 };
+
+export const isSandboxOMD = () => {
+  return (
+    document.location.hostname === 'sandbox.open-metadata.org' ||
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.tsx:504]: @darth-coder00 , can we move out this hostname values to constant.

**Review by Sachin-chaurasiya — APPROVED** [review]
LGTM, one minor comment.

**Review by ShaileshParmar11 — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.tsx (hunk 1 above)
Comment (darth-coder00) [reply openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.tsx:504]: Kept it here as it is going to be replaced by config from api

**Review by harshach — APPROVED** [review]

**Comment by vivekratnavel** [comment]
@darth-coder00 I have a patch available here https://github.com/open-metadata/OpenMetadata/pull/3627 to expose a config API to get the sandbox mode. It would be great to use the API to determine whether sandbox mode is enabled or not. Please take a look and let me know if you have any questions.

**Review by harshach — CHANGES_REQUESTED** [review]
@darth-coder00 incorporate this API to enable this instead of depending on hostname https://github.com/open-metadata/OpenMetadata/pull/3630

**Comment by darth-coder00** [comment]
Blocked on PR #3630

**Review by vivekratnavel — APPROVED** [review]

**Review by harshach — APPROVED** [review]

**Pull request merged**